src/Makefile

```diff
@@ -14,12 +14,16 @@ $(info FC setting: '$(FC)')
 DEBUG ?= 0
 
 # Default to NetCDF build
-NETCDF ?= 1
+NC ?= 1
 
 # Compile flags
 $(info DEBUG setting: '$(DEBUG)')
 ifeq ($(DEBUG), 1)
-  FCFLAGS := -g -Wall -Wextra -Wconversion -Og -pedantic -fcheck=bounds -fmax-errors=5
+  ifeq ($(FC),gfortran)
```

Comment:
I noticed issue with this: your FC must be exactly `ifort` or `gfortran` or you won't get debug flags currently. For example `gfortran-12`, `/usr/bin/gfortran`, won't be recognized. CMake will help make solving this easier and more robust. But for this PR we might want to fix it for the CI case (`gfortran-12`-like), maybe using `notdir` and `findstring`.

Reply:
```diff
 # Compile flags
 $(info DEBUG setting: '$(DEBUG)')
 ifeq ($(DEBUG), 1)
-  ifeq ($(FC),gfortran)
+  ifeq ($(findstring gfortran,$(notdir $(FC))),gfortran)
     FCFLAGS := -g -Wall -Wextra -Wconversion -Og -pedantic -fcheck=bounds -fmax-errors=5 -std=f2003 -fall-intrinsics
   else ifeq ($(FC),ifort)
     FCFLAGS := -g -warn all -check bounds -implicitnone -O0 -error-limit 5
@@ -29,6 +29,7 @@ else ifeq ($(DEBUG), 0)
 else
   $(error invalid setting for DEBUG, should be 0 or 1 but is '$(DEBUG)')
 endif
+$(info FCFLAGS: '$(FCFLAGS)')

 # NETCDF Settings here
 LIBS :=
```

Reply:
Thank you Zach!  Made the change, please let me know!  
```

Reply:
Looks like it is working now!